src/meta/src/hummock/manager/mod.rs

```diff
@@ -997,6 +1011,18 @@ impl HummockManager {
             compact_task.current_epoch_time = Epoch::now().0;
             compact_task.compaction_filter_mask =
                 group_config.compaction_config.compaction_filter_mask;
+            table_to_vnode_partition
+                .retain(|table_id, _| compact_task.existing_table_ids.contains(table_id));
```

Comment:
Is this necessary? If yes, it should be put after L1023. Otherwise, we can mistakenly put non-existing table id in the map.